src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -79,7 +61,7 @@ public enum ModuleCreationOption {
      *
      * @return stream with log
      */
-    public ByteArrayOutputStream getStream() {
+    protected final ByteArrayOutputStream getStream() {
```

Comment:
Just curious, why not allow extension of this?

Reply:
I would say because this a simple getter. If we allowed this to be overriden, then we would always have to call the method instead of the field for our uses in the abstract. Overriding this amounts to providing their own custom stream implementation. Nothing in our area uses it, and I see no uses for it being overridden so hence the `final`.

Truthfully I would prefer to remove this method since it has no real uses, except for WriteTagCheck which is just a flawed check in some aspects.

Reply:
Makes sense to me, I wonder why this is not more common now that you explain it.

Resolved.